controllers/UserController.js

```diff
@@ -126,6 +133,25 @@ module.exports = {
             error: err
           });
         }
+        if (user) {
+          return transport.sendMail({
+            from: 'info@austincodingacademy.com',
+            to: user.username,
+            subject: 'ACA Welcome and Prework',
+            html: "Welcome to Austin Coding Academy! Please register for your desired session, then check your email for additional course materials."
+          }, function (err, info) {
+            if (err) {
+              return res.json(500, {
+                message: 'Error sending confirmation email. Please contact ACA support for additional assistance.'
+              })
+            }
+            else {
+              return res.json(200, {
```

Comment:
We missed this in the review. If this returns the `res`ponse, it will try to return the response again on line 155, resulting in an `Error: Can't set headers after they are sent.` error. See the fix in https://github.com/AustinCodingAcademy/campus-manager/commit/27ac9c0b58f96bc9d5ab851bb46216cb151fe201